core/prob_dist.py

```diff
@@ -32,6 +32,14 @@ def alphabet(self):
     def prob_list(self):
         return [self.prob_dict[s] for s in self.alphabet]
 
+    @classmethod
+    def get_sorted_prob_dist(cls, prob_dict, reverse=True):
+        """
+        Returns:
+            an inheritable ProbabilityDist object with sorted probabilities
```

Comment:
didn't understand the "inheritable ProbabilityDist" part of the comment.what do you mean?